ui/raidboss/data/06-ew/trial/rubicante.txt

```diff
@@ -0,0 +1,127 @@
+# Rubicante
+
+# -ii 8022 7F6C 7CC8 7CCA 7CCF 7CCD 8024 7CBF 8025 7CD0 7CD4 7CD5 7CD6 7CD7 7CD8 7CD1 7CD9 7CBD 7CDB 7CDE 7CE8 7D96 7CC0
+# -it "Rubicante"
+# -p 7CD2:1000 
+
+hideall "--Reset--"
+hideall "--sync--"
+hideall "Arcane Revelation"
+hideall "Infernal Slaughter"
+
+0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+7.0 "--sync--" sync / 14:[^:]*:Rubicante:7CEA:/ window 10,10
+12.0 "Inferno" sync / 1[56]:[^:]*:Rubicante:7CEA:/
+
+# Phase 1 (Purgations & Arch Inferno)
+15.1 "--middle--" sync / 1[56]:[^:]*:Rubicante:7CBC:/
+17.3 "Hope Abandon Ye" sync / 1[56]:[^:]*:Rubicante:7F26:/
+26.7 "Arcane Revelation" sync / 1[56]:[^:]*:Rubicante:7CBE:/
+47.7 "Ordeal of Purgation" sync / 1[56]:[^:]*:Rubicante:7CC4:/ # no spin, straight line
+52.7 "Fiery Expiation" sync / 1[56]:[^:]*:Circle of Purgatory:7CC5:/
+68.9 "Ordeal of Purgation" sync / 1[56]:[^:]*:Rubicante:7CC4:/ # no spin, jagged line
+73.9 "Fiery Expiation" sync / 1[56]:[^:]*:Circle of Purgatory:7CC5:/
+86.9 "Shattering Heat" sync / 1[56]:[^:]*:Rubicante:7CEB:/
+94.1 "--middle--" sync / 1[56]:[^:]*:Rubicante:7CBC:/
+
+102.1 "Arch Inferno" sync / 1[56]:[^:]*:Rubicante:7CC7:/ window 30,30
+102.1 "Inferno Devil" sync / 1[56]:[^:]*:Rubicante:7CC9:/
+105.2 "Infernal Slaughter" sync / 1[56]:[^:]*:Rubicante:7CCE:/
+112.2 "Conflagration" sync / 1[56]:[^:]*:Rubicante:7CCB:/
+115.3 "Infernal Slaughter" sync / 1[56]:[^:]*:Rubicante:7CCE:/
+122.3 "Radial Flagration" sync / 1[56]:[^:]*:Rubicante:7CCC:/
+130.4 "--middle--" sync / 1[56]:[^:]*:Rubicante:7CBC:/
+
+132.8 "Hope Abandon Ye" sync / 1[56]:[^:]*:Rubicante:7F26:/ window 30,30
+142.2 "Arcane Revelation" sync / 1[56]:[^:]*:Rubicante:7CBE:/
+163.1 "Ordeal of Purgation" sync / 1[56]:[^:]*:Rubicante:80E8:/ # middle-disc spin, jagged line (from this point forward)
+171.1 "Fiery Expiation" sync / 1[56]:[^:]*:Circle of Purgatory:7CC5:/
+187.5 "Ordeal of Purgation" sync / 1[56]:[^:]*:Rubicante:80E8:/
+195.5 "Fiery Expiation" sync / 1[56]:[^:]*:Circle of Purgatory:7CC5:/
+201.6 "--untargetable--"
+
+# Adds Phase
+# The adds will intermittently cast various AoEs.
+# All are ground-telegraphed except 7CD4 (Ghastly Torch), a small raid-wide.
+# Because variation in add deaths affects timing, don't sync to them or display timeline.
+201.6 "--sync--" sync / 1[56]:[^:]*:Rubicante:7CBC:/ # boss reposition at edge
+
+# Phase 2 (Flamerake & Sweeping Immolation)
+986.4 "--sync--"  sync / 14:[^:]*:Rubicante:7CD2:/ window 1000,0
+986.4 "--targetable--"
+1000.0 "Blazing Rapture" sync / 1[56]:[^:]*:Rubicante:7CD2:/
+1013.4 "--untargetable--" # cutscene
+1036.4 "--targetable--"
+1048.5 "Inferno (spread)" sync / 1[56]:[^:]*:Rubicante:7CDA:/
+1062.2 "Flamerake (indicator)" sync / 1[56]:[^:]*:Rubicante:7CDD:/
```

Comment:
```suggestion
1062.2 "--sync--" sync / 1[56]:[^:]*:Rubicante:7CDD:/
```

You can take this suggestion or leave it, but given that there's already 1 2 3 in the timeline, maybe the indicator doesn't need to be listed too unless it's a castbar somebody would mit.